mysql-test/suite/galera/t/galera_retry_applying.test

```diff
@@ -0,0 +1,41 @@
+#
+# Test retrying applying of a transaction
+#
+
+--source include/galera_cluster.inc
+
+#
+# Retrying succeeds after one retry event, no error is raised.
+#
+CREATE TABLE t1 (f1 INTEGER PRIMARY KEY DEFAULT 0, f2 char(12));
+
+--connection node_1
+START TRANSACTION;
+INSERT INTO t1 (f1, f2) VALUES (1, 'a'), (2, 'b');
+COMMIT;
+
+START TRANSACTION;
+INSERT INTO t1 (f1, f2) VALUES (3, 'c');
+
+--connection node_2
+SET GLOBAL wsrep_applier_retry_count = 2;
+SET GLOBAL debug_dbug = "+d,innodb_insert_fail:o,/dev/null";
+CALL mtr.add_suppression("WSREP: Event 3 Write_rows_v1 apply failed: 146, seqno 4");
```

Comment:
Better not to include sequence numbers in suppressions, the seqno will not remain the same if the test is run repeatedly or part of full MTR run.

Reply:
Ok, I will fix this as suggested.

Reply:
Fixed a suggested.